regression-test/package.json

```diff
@@ -18,17 +18,14 @@
     "@types/node": "^20",
     "@types/react": "^18",
     "@types/react-dom": "^18",
-    "autoprefixer": "^10.0.1",
-    "cypress": "^13.13.0",
+    "cypress": "^13.17.0",
     "eslint": "^8",
-    "eslint-config-next": "14.1.2",
-    "postcss": "^8",
-    "typescript": "^5",
-    "tailwindcss": "^3"
+    "eslint-config-next": "14.2.26",
+    "typescript": "^5"
   },
   "dependencies": {
     "@instructure/ui": "snapshot",
-    "next": "^14.2.3",
+    "next": "^14.2.26",
     "react": "^18",
     "react-dom": "^18"
```

Comment:
while testing this with SSR, I've updated and simplified our regression test a bit